packages/story-editor/src/components/panels/design/link/link.js

```diff
@@ -201,8 +228,24 @@ function LinkPanel({ selectedElements, pushUpdateForObject }) {
     link.url,
   ]);
 
+  const onChangeRel = useCallback(
+    (value) => {
+      const rel = link.rel.includes(value)
+        ? link.rel.filter((el) => el !== value)
+        : [...link.rel, value];
+      handleChange({ rel }, true);
+    },
+    [handleChange, link]
+  );
+
   const isMultipleUrl = MULTIPLE_VALUE === link.url;
   const isMultipleDesc = MULTIPLE_VALUE === link.desc;
+  const showRelOption =
```

Comment:
We already have `displayMetaFields` for detecting if the URL is set / if it's valid, maybe just use that here as well instead of `link.url`? Or maybe the whole new checkbox area should also be inside the `displayMetaFields` condition when rendering as the rest of the meta fields. Thoughts?

Reply:
Fixed.